api/sheets/get_property.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Get Node Property
+nav_order: 7
+parent: Google Sheets
+grand_parent: API
+---
+
+# Get the values of a given property for a node.
+
+## =DCPROPERTY(dcids, property)
+
+Given a property DCID, together with a string DCID, a row of cells of DCIDs, or a column of cells of DCIDs, get the values of the given property for those DCIDs.
+
+**Arguments**
+*    `dcids` - DCIDs to get the properties of
```

Comment:
maybe property instead of "properties" since here only queries single property

Reply:
Done.